### DISCUSSION TITLE
page navigation with mouse wheel

### DISCUSSION THREAD

**Discussion by reznoa** [gotson/komga#1709] [Ideas]
support page navigation with mouse wheel.
wheel up = prev
wheel down = next

**Comment by w108x** [comment]
Also allow rebinding keys. I like to navigate pages with A and D for example.

**Comment by Tokumi-S** [comment]
Open the console in Chrome browser by pressing F12, then execute the following code (every time you display a Komga page):


window.addEventListener("wheel", function(e) {
    const key = e.deltaY < 0 ? "ArrowRight" : "ArrowLeft"; 
    const event = new KeyboardEvent("keydown", {
        key: key,
        code: key,
        keyCode: key === "ArrowRight" ? 39 : 37,
        which: key === "ArrowRight" ? 39 : 37,
        bubbles: true,
        cancelable: true
    });
    document.activeElement.dispatchEvent(event);
});
